lib/doorkeeper/helpers/controller.rb

```diff
@@ -45,6 +45,8 @@ def get_error_response_from_exception(exception)
                        :invalid_request
                      when Errors::InvalidGrantReuse
                        :invalid_grant
+                     else
+                       exception.message
```

Comment:
Should we put this under a `when Errors` condition? Cause this will be tracking _any_ exception, and we don't want to [`catch Exception`](http://stackoverflow.com/a/10048406/356060). We will want to add test coverage for that, too.